packages/assets-controllers/package.json

```diff
@@ -113,6 +114,7 @@
     "@metamask/preferences-controller": "^17.0.0",
     "@metamask/providers": "^18.1.0",
     "@metamask/snaps-controllers": "^9.19.0",
+    "@metamask/transaction-controller": "^54.0.0",
```

Comment:
In regards to our discussion to use the `BREAKING` keyword does this need to be a peer dependency ?

Reply:
No, it doesn't. 🤦 

Thanks for picking that one up. I'm just using it to import a type, so a devDependency will be enough.

Reply:
Or at this point I'm not clear.

It requires the client to have transaction-controller as it subscribes to an event from it. In the assets-controllers package it is just needed to import a type for that event.

Reply:
Ah, got it. I think it makes sense to keep it as a peer dependency in this case (though @mcmire can confirm).

Reply:
At the very least it is how the rest of the controllers for which we depend just for event subscription are added, but it'd be great to get clarity about it.

Reply:
If we are importing a type from a package, then that type will show up in the TypeScript declaration file, and thus the package must be included in `dependencies` (otherwise TypeScript consumers will need to add that package themselves).

If we are talking to another controller, though, then that controller needs to be added to `devDependencies` + `peerDependencies`, and this rule wins even if we are also using types from that controller package.

So I think this is correct as well.